### PR TITLE
Add Gravity 6.x logging training

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Open source series of workshops delivered by Gravitational services team.
 * [Kubernetes custom resources](crd/crd.md)
 * [Gravity 101](gravity101.md)
 * [Gravity fire drill exercises](firedrills.md)
-* [Gravity logging](logging.md)
+* [Gravity logging](logging-6.x.md)
```
Comment (eldios) [inline README.md]: Should we have to also show up the split difference in the index?

Something like:
```md
* [Gravity logging (Gravity 5.5 and earlier)](logging-5.x.md)
* [Gravity logging (Gravity 6.0 and later)](logging-6.x.md)
```

File: logging-6.x.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+Adapter is not a part of a standard Logrange deployment but rather a Gravity-specific component that provides some additional functionality. It runs as a Deployment:
+
+```bash
+$ kubectl -nkube-system get deploy,pods -lapp=log-collector
+```
+
+The adapter adds the following functionality:
+
+* Serves an HTTP API that allows to query all collected logs. The API is exposed via a Kubernetes service and is used by Gravity Control Panel to provide search functionality. Internally, adapter uses Logrange’s [LQL](https://www.logrange.io/docs/lql.html) to query the data.
+* It is also responsible for updating log forwarder configurations when they are created or deleted by users.
```
Comment (a-palchikov) [inline logging-6.x.md]: created or modified

File: logging-6.x.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+$ kubectl -nkube-system get configmaps/log-forwarders -oyaml
+```
+
+Then, Logrange adapter picks up the change and updates the forwarder configuration:
+
+```bash
+$ kubectl -nkube-system get configmaps/lr-forwarder -oyaml
+```
+
+After that, the forwarder starts sending the logs into the configured destination using the syslog protocol.
```
Comment (a-palchikov) [inline logging-6.x.md]: nit: sending the logs **to** the configured destination

File: logging-6.x.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+```bash
+planet$ ls -l /var/log/containers/
+total 180
+lrwxrwxrwx 1 root root 66 Jan 15 00:24 bandwagon-6c4b...-lqbll_kube-system_bandwagon-2641....log -> /var/log/pods/0b8e.../bandwagon/1.log
+```
+
+## Logrange Components
+
+Logrange consists of the following main components: collector, aggregator and forwarder. These components can be seen in the following diagram.
```
Comment (aelkugia) [inline logging-6.x.md]: Does it make sense to link to diagram? (adding source) - I may have missed it

Comment (r0mant) [reply logging-6.x.md]: Oh lol. I forgot to actually include it.

**Pull request merged**